### DISCUSSION TITLE
Imported models for Special Task

### DISCUSSION THREAD

**Discussion by DerryLin-Tamu** [SimDynamX/SpaceTeamsPRO#75] [2024 Lunar SAR Competition Q&A] [answered]
Will there be a tutorial on how to import models into the pre-made simulations, and are the current models in the simulations from blender or cad?

**Comment by connorjak** [comment, marked as answer]
https://github.com/SimDynamX/SpaceTeamsPro/wiki/STU-Lunar-SAR-Competitor-Setup-&-Learning#lesson-5-model-import-optional Here is a link to where our latest model import tutorial is. It is not using our latest and greatest model import code, but the methods in there should still work. We don't have docs on the new `Object_Loaded` entity type model import yet, but if you wanted to see it in use, both LTVs are using runtime-imported models using that entity type.
